targets/TARGET_NORDIC/TARGET_NRF5/us_ticker.c

```diff
@@ -74,11 +77,19 @@ void COMMON_RTC_IRQ_HANDLER(void)
 
     rtc_ovf_event_check();
 
+    if (m_common_sw_irq_flag & US_TICKER_SW_IRQ_MASK) {
+        m_common_sw_irq_flag &= ~US_TICKER_SW_IRQ_MASK;
```

Comment:
@0xc0170 @mprse
What about moving `m_common_sw_irq_flag &= ~US_TICKER_SW_IRQ_MASK;` to `us_ticker_clear_interrupt` function 
and flatten it to single if as it was before ?

Reply:
looking at the codeflow, clear_interrupt is called right in the ticker_irq_handler, thus this could be moved there. 
Do not follow this one ` flatten it to single if as it was before` ?

Reply:
I meant single `if` like below. Single call of `us_ticker_irq_handler` will execute all outdated events
```
    if ((m_common_sw_irq_flag & US_TICKER_SW_IRQ_MASK) || nrf_rtc_event_pending(COMMON_RTC_INSTANCE, US_TICKER_EVENT)) {
        us_ticker_irq_handler();
    }
```

Reply:
+1